website/static/js/registrationUtils.js

```diff
@@ -475,32 +478,29 @@ RegistrationEditor.prototype.init = function(draft) {
     var metaSchema = draft.metaSchema;
 
     var schemaData = {};
-    if(draft) {
+    if (draft) {
         schemaData = draft.schemaData || {};
     }
 
     var questions = self.flatQuestions();
     $.each(questions, function(i, question) {
         var val = schemaData[question.id];
-        if(val) {
-            if(question.type === 'object') {
+        if (val) {
+            if (question.type === 'object') {
```

Comment:
It's probably too late to change the schema (where `type` is a question field and this is one of the allowed values)... but I definitely found this nomenclature confusing at first.